br/pkg/lightning/config/config.go

```diff
@@ -1361,17 +1361,10 @@ func (c *Conflict) adjust(i *TikvImporter, l *Lightning) error {
 		return common.ErrInvalidConfig.GenWithStack(
 			"unsupported `%s` (%s)", strategyConfigFrom, c.Strategy)
 	}
-	if c.Strategy != NoneOnDup {
-		if i.ParallelImport && i.Backend == BackendLocal && c.PrecheckConflictBeforeImport {
-			return common.ErrInvalidConfig.GenWithStack(
-				`%s cannot be used with tikv-importer.parallel-import and tikv-importer.backend = "local" and conflict.precheck-conflict-before-import = true`,
-				strategyConfigFrom)
-		}
-		if !strategyFromDuplicateResolution && i.DuplicateResolution != NoneOnDup {
-			return common.ErrInvalidConfig.GenWithStack(
-				"%s cannot be used with tikv-importer.duplicate-resolution",
-				strategyConfigFrom)
-		}
+	if !strategyFromDuplicateResolution && c.Strategy != NoneOnDup && i.DuplicateResolution != NoneOnDup {
```

Comment:
Now， we will not distinct whether it is logical import?

Reply:
I removed the check of 
```
if c.Strategy != NoneOnDup {
		if i.ParallelImport && i.Backend == BackendLocal && c.PrecheckConflictBeforeImport {
```
Since now we can support parallel import with preprocess conflict detection for local backend.

Originally this was not supported because preprocess and post-import conflict detection are not allowed to be used together, so only conducting preprocess conflict detection may still result in conflicts in the scenarios of parallel import.

Reply:
So now we support preprocess + post-import conflict， post-import conflict。 the post-import conflict will always enabled, right?

Reply:
yes